hawc/apps/common/api.py

```diff
@@ -1,15 +1,85 @@
+from django.conf import settings
 from django.db import models
 from django.shortcuts import get_object_or_404
-from rest_framework import exceptions, filters, mixins, permissions, viewsets
+from django.utils.encoding import force_str
+from rest_framework import exceptions, filters, mixins, permissions, status, viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
 from rest_framework.throttling import UserRateThrottle
-from rest_framework_extensions.mixins import ListUpdateModelMixin
 
 from ..assessment.api import DisabledPagination, get_assessment_from_query
 from .helper import try_parse_list_ints
 
 
+class ListUpdateModelMixin:
```

Comment:
move this into it's own file. can you add a comment where you had to change the original code for 3.1 compatibility?

Reply:
ideally, it woudl be in `hawc/apps/common/api/mixins.py`, but that means you'll need to do a little more refactoring of the api.py. We can import the things in the `api/__init__.py` so all the external imports would still work. do you think that's a good plan